kratos/utilities/nurbs_utilities/snake_sbm_utilities.cpp

```diff
@@ -0,0 +1,838 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "snake_sbm_utilities.h"
+
+namespace Kratos
+{
+
+    void SnakeSbmUtilities::CreateTheSnakeCoordinates(ModelPart& rIgaModelPart,
+                                                    ModelPart& rSkinModelPartInnerInitial,
+                                                    ModelPart& rSkinModelPartOuterInitial, 
+                                                    ModelPart& rSkinModelPart, 
+                                                    int rEchoLevel, 
+                                                    Vector& knotVectorU, 
+                                                    Vector& knotVectorV,
+                                                    const Parameters mParameters) { 
```

Comment:
```suggestion
void SnakeSbmUtilities::CreateTheSnakeCoordinates(
    ModelPart& rIgaModelPart,
    ModelPart& rSkinModelPartInnerInitial,
    ModelPart& rSkinModelPartOuterInitial, 
    ModelPart& rSkinModelPart, 
    int rEchoLevel, 
    Vector& knotVectorU, 
    Vector& knotVectorV,
    const Parameters mParameters)
{ 
```
Please follow our standard. Do so for the rest of methods.